android/plugins/sharedpreferences/SharedPreferencesSonarPlugin.java

```diff
@@ -23,6 +23,21 @@
 
   private SonarConnection mConnection;
   private final SharedPreferences mSharedPreferences;
+  private final SharedPreferences.OnSharedPreferenceChangeListener onSharedPreferenceChangeListener = new SharedPreferences.OnSharedPreferenceChangeListener() {
+    @Override
+    public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, String key) {
+      if (mConnection != null) {
```

Comment:
I know that this was copied over, but can we early return here instead of indenting the entire block?

```
if (mConnection == null) {
  return;
}
```